pkg/cmd/registry/describe/describe.go

```diff
@@ -20,22 +23,24 @@ type options struct {
 	name         string
 	outputFormat string
 
-	IO         *iostreams.IOStreams
-	Config     config.IConfig
-	Connection factory.ConnectionFunc
-	localizer  localize.Localizer
-	Context    context.Context
+	IO             *iostreams.IOStreams
+	Config         config.IConfig
+	Connection     factory.ConnectionFunc
+	localizer      localize.Localizer
+	Context        context.Context
+	ServiceContext servicecontext.IContext
 }
 
 // NewDescribeCommand describes a service instance, either by passing an `--id flag`
 // or by using the service instance set in the config, if any
 func NewDescribeCommand(f *factory.Factory) *cobra.Command {
```

Comment:
Will we need a new `--context` flag so you can describe from different contexts?

Reply:
I think that will make things look more complicated, an user should be able to see the instance set in current context and other instances by ID. Not sure if adding a `--context` flag will be an improvement, wdyt?

Reply:
Agree. Context should be contained to the rhoas context commands.